projekt_helm/fiszki/templates/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backend-deployment
+  namespace: {{.values.namespace.name}}
+  labels:
+    app: {{.values.image.type}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.values.image.type}}
+  template:
+    metadata:
+      labels:
+        app: {{.values.image.type}}
+    spec:
+      containers:
+      - name: backend
+        image: {{.values.image.image}}
+        imagePullPolicy: {{.values.image.imagePullPolicy}}
```

Comment:
Odwołanie do pliku `values.yaml` jest z dużej. Przynajmniej u mnie dopóki tego nie zmienię to się wywala ;)